mirest/stylist-api
# #165026938 Configure application to use GraphQL

#### What does this PR do?
 - It configures the application to allow use of graphQL

#### Description of Task to be completed?
The task requires us to be able to use graphQL queries and mutations.

#### How can this be tested manually?
- Check out to `ch-configure-graphql-165026938`.
- Start the app and run the following query;
```
query{
  users{
    id
    email
    phoneNumber
    isActive
  }
}
```
- Results returned will be similar to those shown in the screenshot

#### Screenshots
<img width="1189" alt="Screenshot 2019-04-04 at 11 06 28" src="https://user-images.githubusercontent.com/40039818/55539676-e14f5a00-56c9-11e9-9dd2-01d9259c3a7b.png">

#### Context
Using the insonmia app to run the query mentioned above

#### Pivotal Tracker Story
- [#165026938](https://www.pivotaltracker.com/story/show/165026938)
